.golangci.yml

```diff
@@ -1,3 +1,5 @@
+run:
+  timeout: 5m
```

Comment:
Thanks for bumping this up. We can keep an eye on it to see what appropriate timeout should be with using it through GitHub Actions.